### PR TITLE
Fixing issues with missing icons + active color

### PR DESCRIPTION
**Pull request by michalolborski**
missing icons after HA 2023.2.0, issues:
https://github.com/thomasloven/lovelace-template-entity-row/issues/100

wrong active color after HA 2023.1.0, issues:
https://github.com/thomasloven/lovelace-template-entity-row/issues/95
https://github.com/thomasloven/lovelace-template-entity-row/issues/94

### CODE REVIEW

**Comment by Erelen-Laiquendi** [comment]
Doesn't work without `entity` parameter :( It shows blank row.

**Comment by ildar170975** [comment]
> Doesn't work without `entity` parameter

This is my case: https://github.com/thomasloven/lovelace-template-entity-row/issues/100#issuecomment-1423381429
an icon is NOT displayed if no entity is defined:
```
      type: custom:template-entity-row
      state: yyy
      name: xxx
      icon: mdi:car
```
Do you mean that this PR does not solve this particular issue?
Cannot test it myself... Need to have a compiled js-file to install.

**Comment by michalolborski** [comment]
@Erelen-Laiquendi , @ildar170975 - hope it explains everything ;)

image

this is how this component is working after this PR

**Comment by ildar170975** [comment]
@michalolborski 
Seems to be working!
Could you share a compiled js-file?

File: src/main.ts [hunk 1]
```diff
@@ -82,11 +82,7 @@ class TemplateEntityRow extends LitElement {
   }
 
   render() {
-    const base = this.hass.states[this.config.entity];
-    const entity = (base && JSON.parse(JSON.stringify(base))) || {
-      entity_id: "light.",
-      attributes: { icon: "no:icon" },
-    };
+    const entity = this.hass.states[this.config.entity];
```
Comment (TheHolyRoger) [inline src/main.ts:85]: @michalolborski I don't think you need this change? This seems to cause issues when leaving the entity blank and works fine without this change

**Comment by TheHolyRoger** [comment]
@michalolborski Can I suggest you stick your own fork on HACS? If you're not happy to maintain it maybe I will instead... @thomasloven seems to have dropped this one I think?

**Comment by michalolborski** [comment]
> @michalolborski Can I suggest you stick your own fork on HACS? If you're not happy to maintain it maybe I will instead... @thomasloven seems to have dropped this one I think?

you can take over if you like :)

**Comment by TheHolyRoger** [comment]
Fixed in my fork here, feel free to add it as a custom repo in HACS: https://github.com/TheHolyRoger/lovelace-template-entity-row

Remove this one first then add mine 👍

**Pull request closed without merging**